src/main/java/me/zeroeightsix/kami/module/modules/movement/ElytraFlight.kt

```diff
@@ -182,38 +202,134 @@ class ElytraFlight : Module() {
         } else elytraDurability = 0
 
         /* Holds player in the air if run out of durability */
-        if (!mc.player.onGround && outOfDurability && elytraDurability <= 1) {
-            event.cancel()
-            mc.player.setVelocity(0.0, -0.01, 0.0)
+        if (!mc.player.onGround && elytraDurability <= 1 && outOfDurability) {
```

Comment:
elytraDurability isn't actually changed anywhere, other then line 202 (but it stays 0 the entire time)

Reply:
Check line 187

Reply:
??
![dominikaaaa org-2020-07-08-20-30-03](https://user-images.githubusercontent.com/17222512/86983418-d2234b80-c159-11ea-8745-043386e78353.png)

Reply:
The new line 187 not the old one

Reply:
Oh I see.